tests/unit/sprite/test_sprite_collision.py

```diff
@@ -7,16 +7,22 @@ def test_sprites_at_point():
 
     coin_list = arcade.SpriteList()
     sprite = arcade.SpriteSolidColor(50, 50, color=arcade.csscolor.RED)
+    # an adjacent sprite with the same level horizontal bottom edge
+    sprite2 = arcade.SpriteSolidColor(50, 50, center_x=50, center_y=0, color=arcade.csscolor.RED)
+    
     coin_list.append(sprite)
+    coin_list.append(sprite2)
 
-    # print()
```

Comment:
Not sure why these comments are here

Reply:
Just remove if they don't make sense. It means they have low value.